### ISSUE TITLE
Probably not a bug, just asking

### ISSUE THREAD

**Issue by PaulGrandperrin** [dtolnay/semver#171]
I was fuzzing semver with this [code](https://github.com/rust-fuzz/targets/blob/master/semver/version_req_read_write_read.rs) when it failed.

Here is the error: [playground](https://play.rust-lang.org/?gist=cf9f4e685b12b124e385640da3411a83&version=stable)

I'm not sure this is a legitimate issue. If not, I'll remove this fuzz target.

**Comment by killercup** [comment]
Similar issue with wildcard in weird position, also from fuzzing, with `9,88.*-76`. [demo](https://play.rust-lang.org/?gist=51b368d12688ef143a0cd21f8311e247&version=stable)

**Comment by dtolnay** [comment]
The issue with `>=1.*.3` is fixed in #234.

```rust
fn main() {
    println!("{}", semver::VersionReq::parse(">=1.*.3").unwrap_err());
}
```

```console
unexpected character after wildcard in version req
```

**Comment by dtolnay** [comment]
The second issue also appears to be fixed.

```rust
fn main() {
    println!("{}", semver::VersionReq::parse("9,88.*-76").unwrap_err());
}
```

```console
expected comma after minor version number, found '-'
```
